### ISSUE TITLE
Styled Navigation Bar

### ISSUE THREAD

**Issue by cpdevelops** [cpdevelops/simple.css#39]
Component requested : Navigation bar similar to the one in [Bootstrap](getbootstrap.com). Medium difficulty issue for Hacktoberfest. Please notify me if you would like to take this over, and please read the README and CONTRIBUTING files before starting. Thanks and happy coding!

**Comment by cpdevelops** [comment]
Duplicate of #40

**Comment by dvlden** [comment]
Your reference is not related to the navigation bar. You did a small mistake here @cpdevelops

**Comment by cpdevelops** [comment]
Yes I did @dvlden haha. I thought I was on the right issue but I guess I have multiple tabs open with different issues. Reopening now.

**Comment by susannahfisher** [comment]
Hi! I'd like to work on the nav bar. I read the README, but don't see the CONTRIBUTING fils in your list. Let me know if this is something you'd like me to take on. Cheers!

**Comment by dvlden** [comment]
@susannahfisher - You may find CONTRIBUTING file over here...
[CONTRIBUTING.md](https://github.com/cpdevelops/simple.css/blob/master/.github/CONTRIBUTING.md)

**Comment by susannahfisher** [comment]
Awesome, thanks. I've read it all and I'd can work on this for Hacktober, if you'd like.

**Comment by dvlden** [comment]
I'm sure @cpdevelops will assign you this when he's available. You could start working on it, I assume?

**Comment by cpdevelops** [comment]
You can definitely get started @susannahfisher. I will mark this issue as assigned, and I will be watching for your PR

**Comment by soumam007** [comment]
@susannahfisher  **if** you arent working on this , i want to contribute :) or else you can continue

**Comment by susannahfisher** [comment]
@soumam007 I'm working on it - 👍

**Comment by soumam007** [comment]
@susannahfisher are you still working on it?

**Comment by cpdevelops** [comment]
@susannahfisher Please respond to if you are still working on this project and are almost done, because if not I can go ahead and assign someone else.

**Comment by susannahfisher** [comment]
Go ahead and assign it to someone else - we had an emergency here & I can't prioritize this. Thanks.

**Comment by cpdevelops** [comment]
Perfectly fine. Thank you for telling me.

Removing the assignment on this issue.

**Comment by soumam007** [comment]
@cpdevelops I want to work on it . 
@susannahfisher no issues you can add on to it later :)

**Comment by cpdevelops** [comment]
Ok @soumam007 . Go ahead and get to work, I will add the assigned label back onto this issue.
